pkg/kube/client.go

```diff
@@ -57,19 +57,25 @@ type Client struct {
 	Log     func(string, ...interface{})
 }
 
+var addToScheme sync.Once
+
 // New creates a new Client.
 func New(getter genericclioptions.RESTClientGetter) *Client {
 	if getter == nil {
 		getter = genericclioptions.NewConfigFlags(true)
 	}
 	// Add CRDs to the scheme. They are missing by default.
-	if err := apiextv1.AddToScheme(scheme.Scheme); err != nil {
-		// This should never happen.
-		panic(err)
+	var errv1, errv1beta1 error
+	addToScheme.Do(func() {
+		errv1 = apiextv1.AddToScheme(scheme.Scheme)
```

Comment:
if we're going to `panic()`, it should be called immediately before calling another function. Can you please re-write this to the following:

```
addToScheme.Do(func() {
    if err := apiextv1.AddToScheme(scheme.Scheme); err != nil {
        // This should never happen.
        panic(err)
    }
    if err := apiextv1beta1.AddToScheme(scheme.Scheme); err != nil {
        // This should never happen.
        panic(err)
}
```

Reply:
Done that way :) :heavy_check_mark: